components/list/list-item-button-mixin.js

```diff
@@ -4,13 +4,30 @@ import { getUniqueId } from '../../helpers/uniqueId.js';
 import { ListItemMixin } from './list-item-mixin.js';
 
 export const ListItemButtonMixin = superclass => class extends ListItemMixin(superclass) {
+	static get properties() {
+		return {
+			/**
+			 * Disables the primary action button
+			 * @type {boolean}
+			 */
+			buttonDisabled : { type: Boolean, attribute: 'button-disabled' }
+		};
+	}
 
 	static get styles() {
 
 		const styles = [ css`
-			:host {
+			:host(:not([button-disabled])) {
 				--d2l-list-item-content-text-color: var(--d2l-color-celestine);
 			}
+			:host([button-disabled][_hovering-primary-action]) .d2l-list-item-content,
+			:host([_focusing-primary-action]) .d2l-list-item-content {
```

Comment:
```suggestion
			:host([button-disabled][_focusing-primary-action]) .d2l-list-item-content {
```
I think we only want to disable the focus styles if the button is disabled

Reply:
Focusing has to do with keyboard navigation. Hovering is the mouse navigation having a focus on the element.  Do we want the color change and underlining style when mouse is over the primary action? I think we need to disable for both.

Reply:
Yep, I think we want to keep the focus style for active buttons and disable it only for disabled buttons. That's the current behaviour in both list item links and list item buttons
Current:
![Screen Shot 2023-01-26 at 4 29 36 PM](https://user-images.githubusercontent.com/12901943/214955136-9f9019b8-fdbe-43d2-8dcf-84827c42d018.png)
This branch:
![Screen Shot 2023-01-26 at 4 29 49 PM](https://user-images.githubusercontent.com/12901943/214955139-bdc6d640-9894-42d0-b6ab-39a0c5f9c335.png)